src/acquisition/covid_hosp/facility/database.py

```diff
@@ -40,7 +40,7 @@ class Database(BaseDatabase):
       Columndef('ccn', 'ccn', str),
       Columndef('city', 'city', str),
       Columndef('fips_code', 'fips_code', str),
-      Columndef('geocoded_hospital_address', 'geocoded_hospital_address', str),
+      Columndef('geocoded_hospital_address', 'geocoded_hospital_address', Utils.limited_string_fn(32)),
```

Comment:
is it worth doing this for the [other string fields](https://github.com/cmu-delphi/delphi-epidata/blob/a55cc108fd1f4ac77664f90a87069f418e47037d/src/ddl/covid_hosp.sql#L1070-L1082) too?

Reply:
This particular function isn't going to stick around; we need something different for geocode-specific truncation.

I'm wavering on whether to explicitly encode the varchar lengths in the Python code for the other string fields, since that adds to the level of detail we have to track in both places (.sql and database.py)